layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,138 @@
+      padding-bottom: 30px;
+    }
+
+    ul.humans > li {
+      list-style: none;
+      font-size: 1.7rem
+    }
+
+    h1 {
+      font-size: 5rem
+    }
+
+    .center {
+      margin: 0 auto
+    }
+
+    .space {
+      padding-top: 4rem;
+      padding-bottom: 2rem;
+    }
+
+    footer {
+      margin-bottom: 2rem;
+    }
+  </style>
+</head>
+
+<body>
+  <h1 class="center space">#BlackLivesMatter</h1>
+  <section id="lives" class='center'>
+    <ul class="humans">
+      <li>George Floyd</li>
+      <li>Ahmaud Arbery</li>
+      <li>Mike Ramos</li>
+      <li>Tamir Rice</li>
+      <li>Michael Brown</li>
+      <li>Freddie Gray</li>
+      <li>Eric Garner</li>
+      <li>Philando Castile</li>
+      <li>Anton Sterling</li>
+      <li>Delrawn Small</li>
+      <li>Sandra Bland</li>
+      <li>Botham Jean</li>
+      <li>Trayvon Martin</li>
+      <li>Jamar Clark</li>
+      <li>Laquan McDonald</li>
+      <li>Antwon Rose II</li>
+      <li>Keith Childress</li>
+      <li>Walter Scott</li>
+      <li>Alton Sterling</li>
+      <li>Stephon Clark</li>
+    </ul>
+    <p><a href="https://mappingpoliceviolence.org/unarmed">and</a> <a href="">far</a>, <a href="">too</a> <a href="">many</a> <a href="">more</a>.</p>
+  </section>
+
+  <section>
+    <p>Where you can donate to with your wallet:</p>
+    <ul>
+      <li>Donate to the <a href="https://www.gofundme.com/f/georgefloyd">Official George Floyd Memorial Fund</a>.</li>
+      <li>Donate to the <a href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a> fundraiser for Ahmaud Arbery.</li>
+      <li>Debate to the <a href="https://www.gofundme.com/f/justice-for-mike-ramos">Justice for Mike Ramos</a> fundraiser.</li>
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you. Each one of us in the list below has put out a message (linked) asserting our support and willingness to help - feel free to ping us in these threads, via open DMs on the linked accounts, or by sending us an email. We are here for you.</p>
+
+    <section id="we-are-here-for-you">Tierney Cyren (<a href='https://twitter.com/bitandbang/status/1267893948149911556?s=20'>message of support</a>, <a href='https://twitter.com/bitandbang'>twitter</a>)</section>
```

Comment:
I opened a couple of PRs in your original repo for suggested edits here.

Reply:
I a hard -1 on an empty statement of "we're here for you" with no actual demonstration of being there.